zzzeeksphinx/themes/zsbase/layout.mako

```diff
@@ -84,6 +84,8 @@ withsidebar = bool(toc) and (
         <link rel="prev" title="${prevtopic['title']|util.striptags}" href="${prevtopic['link']|h}" />
     % endif
     <!-- end layout.mako headers -->
+    <script type="text/javascript" id="documentation_options" data-content_root="${ pathto('', 1) }" src="${ pathto('_static/documentation_options.js', 1) }"></script>
```

Comment:
I think this is still wrong.

`data-content_root` should be an attribute of the top-level `<html>` node, not of the `<script>` node.

See for reference:
- https://github.com/sphinx-doc/sphinx/blob/v7.2.6/sphinx/themes/basic/static/searchtools.js#L65 (`documentElement` is the `<html>` element)
- https://github.com/sphinx-doc/sphinx/commit/8e730ae303ae686705ea12f44ef11da926a87cf5

Reply:
that's what's in the actual base template they have so I agree that's where it should go

Reply:
which of course is inconvenient here because SQLAlchemy's site has the actual base in a different repo, so will try to reverse engineer this into something platform agnostic